site/_docs/client_reference.md

```diff
@@ -193,3 +193,52 @@ on-hover images for the permalink, but oh well.
 : _Default_: `false`.
 
 : _Required_: No.
+
+<strong><a name="use_client_side_lb" href="#use_client_side_lb">use_client_side_lb</a></strong>
+
+: _Description_: Enables the client side load-balancing.
+: _Default_: `false`.
+
+: _Required_: No.
+
+<strong><a name="lb_urls" href="#lb_urls">lb_urls</a></strong>
+
+: _Description_: List of URLs in a comma separated format, for example "URL1,URL2...URLn", to be used by the client side
+load balancer. Depending on the load balancing strategy, load balancer selects one of the URLs from the list.
+
+: _Default_: `null`.
+
+: _Required_: No.
+
+<strong><a name="lb_strategy" href="#lb_strategy">lb_strategy</a></strong>
+
+: _Description_: The load balancing strategy to be used by the client side load balancer. It must be a fully qualified
+Java class name which implements `org.apache.calcite.avatica.ha.LBStrategy`. Out of the box we have provided two
+implementations `org.apache.calcite.avatica.ha.RandomSelectLBStrategy` and
+`org.apache.calcite.avatica.ha.RoundRobinLBStrategy` which you can use as load balancing strategies.
```

Comment:
nit:
The load balancing strategy to be used by the client side load balancer. It must be a fully qualified
Java class name which implements `org.apache.calcite.avatica.ha.LBStrategy`.
Two implementations are provided: `org.apache.calcite.avatica.ha.RandomSelectLBStrategy` and
`org.apache.calcite.avatica.ha.RoundRobinLBStrategy` .

Reply:
Yes. Here are my thoughts.
Knox is centralized (all client send requests to KNOX) mechanism where as we implemented distributed (all client independently do the LB). For centralized mechanism, RoundRobin makes more sense. But for distributed approach Random select makes more sense. 
To give an example, if we have some clients C1, C2, C3 interacting with DB at the same time using DB URLs DB1,DB2,DB3. So if the clients are doing transactions concurrently it's likely that they can overload Perticular DB server ( e.g. all C1, C2, C3 pickup first DB URL and connect to i). With Random RandomSelect algorithm there is less probability that all clients connect to same DB and overload it.

Reply:
Done. Thanks.